### PR TITLE
Adds TGUI For Admin Verbs

### PR DESCRIPTION
**Pull request by Wisemonster**
[QoL] [TGUI]

## About the PR 
Adds TGUI to various admin verbs.

## Why's this needed? 
The admins can have a little bit of TGUI, as a treat.

### CODE REVIEW

File: code/modules/admin/admin_verbs.dm [hunk 1]
```diff
@@ -1599,7 +1599,9 @@ var/list/fun_images = list()
 	set desc = "Create a custom object spewing grenade"
 	ADMIN_ONLY
 
-	var/new_grenade = alert("Use the new thing throwing grenade?", "Cool new grenade?", "Yes", "No") == "Yes"
+	var/new_grenade = tgui_alert(usr, "Use the new thing throwing grenade?", "Cool new grenade?", list("Yes", "No"))
+	if(new_grenade == "No") //why wasn't this here before?
```
Comment (frawhst) [inline code/modules/admin/admin_verbs.dm:1603]:
```suggestion
	if(new_grenade == "No")
```
lol that is dumb
comment isn't needed though

File: code/modules/admin/admin_verbs.dm [hunk 2]
```diff
@@ -1178,14 +1178,14 @@ var/list/fun_images = list()
 	else
 		delay = inputDelay
 
-	var/confirm2 = alert("Make everyone invisible? (Literally every mob)", "Invisible Mobs?", "No", "Yes")
+	var/confirm2 = tgui_alert(usr, "Make everyone invisible? (Literally every mob)", "Invisible Mobs?", list("No", "Yes"))
```
Comment (frawhst) [inline code/modules/admin/admin_verbs.dm:1181]: Can you swap the order of all the buttons in this command? Standard is cancel on the right, but all these are `Yes` on the right.

Comment (Wisemonster) [reply code/modules/admin/admin_verbs.dm:1181]: Sure, you want me to do it for the others as well? There's about 4 others that also are doing this.

Comment (frawhst) [reply code/modules/admin/admin_verbs.dm:1181]: Yeah, all of them would be good.

**Review by frawhst — APPROVED** [review]

**Pull request merged**